pylutron/__init__.py

```diff
@@ -29,6 +32,290 @@
   socket.timeout,
 )
 
+class Processor(object):
+  """ Encapsulates the specific communication protocols associated with a Lutron Processor. 
```

Comment:
no space after """

Add a blank line between the first line and rest of description. Applies to rest of your change.